### PR TITLE
Homework 003 done.

### PR DESCRIPTION
**Pull request by ghost**
Task003 done.

### CODE REVIEW

File: src/main/java/com/epam/university/java/core/task003/FilteringConditionImpl.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.epam.university.java.core.task003;
+
+public class FilteringConditionImpl implements FilteringCondition {
+    @Override
+
+
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task003/FilteringConditionImpl.java]: Don't separate code and annotations

File: src/main/java/com/epam/university/java/core/task003/Task003Impl.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
+package com.epam.university.java.core.task003;
+
+
+import java.util.*;
+import java.util.stream.Stream;
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task003/Task003Impl.java]: Don't use wildcards

File: src/main/java/com/epam/university/java/core/task003/FilteringConditionImpl.java (hunk 1 above)
Comment (ghost) [reply src/main/java/com/epam/university/java/core/task003/FilteringConditionImpl.java]: Corrected.

File: src/main/java/com/epam/university/java/core/task003/Task003Impl.java (hunk 2 above)
Comment (ghost) [reply src/main/java/com/epam/university/java/core/task003/Task003Impl.java]: Ok.

**Pull request merged**